gutenberg.php

```diff
@@ -29,6 +29,13 @@ function the_gutenberg_project() {
 	global $post_type_object;
 	?>
 	<div class="block-editor gutenberg">
+		<noscript>
+			<div class="error" style='margin-top:32px'><p>
```

Comment:
The admin bar has different heights depending on the viewport.

Reply:
The margin was simply to give some spacing from the bottom of the admin bar, so the error message isn't squished up against the top.